src/content/en/fundamentals/index.md

```diff
@@ -10,22 +10,48 @@ nav.devsite-page-nav, .devsite-rating-container, .page-title {display:none;}
 
 # Web Fundamentals {: .page-title }
 
-## Web Push Notifications, a new way to engage users {: .hide-from-toc  }
-
-![Example Push Notification](/web/images/common/push-notification-16x9.png){: .attempt-right }
-
-Web push notifications makes it easy to re-engage with users by
-showing relevant, timely, and contextual notifications to the user, 
-even when the browser is closed!
-
-Web push notifications allow users to opt-in to timely updates from
-sites they love and allow you to effectively re-engage them with
-customized, relevant content.
+<div class="attempt-left">
+  <h2>Web Push</h2>
+  <img src="/web/images/common/push-notification-16x9.png" class="attempt-right">
+  <p>
+    Web push notifications makes it easy to re-engage with users by
+    showing relevant, timely, and contextual notifications to the user, 
+    even when the browser is closed!
+  </p>
+  <p>
+    Web push notifications allow users to opt-in to timely updates from
+    sites they love and allow you to effectively re-engage them with
+    customized, relevant content.
+  </p>
+  <p>
+    The Push API and Notification API open a whole new set of
+    possibilities for you to re-engage with your users.
+  </p>
+  <a href="engage-and-retain/push-notifications/" class="button button-primary">
+    Get Started
+  </a>
+</div>
 
-The Push API and Notification API open a whole new set of
-possibilities for you to re-engage with your users.
+<div class="attempt-right">
+  <h2>Building Web Components</h2>
+  <img src="/web/images/common/webcomponents-logo.png" class="attempt-right">
```

Comment:
Two options for handling this.
1. Put your own custom responsive class on it
2. Ignore for now, this page will get a refresh in the next 4-6 weeks.